README.md

```diff
@@ -550,6 +550,9 @@ config :my_app, MyApp.Repo,
 config :fun_with_flags, :persistence,
   adapter: FunWithFlags.Store.Persistent.Ecto,
   repo: MyApp.Repo
+
+# Optional configuration to specify the database table name. Default table name is "fun_with_flags_toggles".
+config :fun_with_flags, table_name: "your_table_name"
```

Comment:
@tompave you had mentioned this code block, but I think it makes sense in context? 
![image](https://user-images.githubusercontent.com/6934825/76358469-e06a7380-62ef-11ea-97fb-d0d9949fcc54.png)